index/scorch/snapshot_index.go

```diff
@@ -46,6 +51,8 @@ var reflectStaticSizeIndexSnapshot int
 func init() {
 	var is interface{} = IndexSnapshot{}
 	reflectStaticSizeIndexSnapshot = int(reflect.TypeOf(is).Size())
+	lb1, lbErr1 = lev2.NewLevenshteinAutomatonBuilder(1, true)
```

Comment:
another way of doing this is panic()'ing if the lev2 returns error here in init()?  that way, later code doesn't need to check for lb1 != nil